examples/nextjs-live-cursors-advanced/components/LiveCursors.tsx

```diff
@@ -26,7 +26,10 @@ export default function LiveCursors({ cursorPanel }: Props) {
   /**
    * Return all the other users in the room and their presence (a cursor position in this case)
    */
-  const others = useOthers();
+  const others = useOthersMapped((other) => ({
+    cursor: other.presence.cursor,
+    info: other.info,
+  }));
```

Comment:
You are creating a new object here every time, so you'll need a `shallow` for this one.